src/lib/commandTypes.ts

```diff
@@ -189,6 +189,13 @@ export type CommandArgumentConfig<
             commandBarContext: ContextFrom<typeof commandBarMachine>,
             machineContext?: C
           ) => string)
+      validation?: ({
+        data,
+        context,
+      }: {
+        data: any
+        context: CommandBarContext
+      }) => Promise<boolean | string>
```

Comment:
The changes here are about enabling the 'kcl' arg type to run a validation function.